setup.py

```diff
@@ -44,16 +44,32 @@ def _minimal_ext_cmd(cmd):
             GIT_REVISION = out.decode('utf-8').strip()
         except OSError:
             GIT_REVISION = "Unknown"
+        try:
+            cmd = ['git', 'config', '--get', 'remote.origin.url']
+            out = _minimal_ext_cmd(cmd).decode('utf-8').strip()
+            start_idx = 0
+            for prefix in [b'git@github.com:', b'git://github.com/',
+                           b'http://github.com/', b'https://github.com/',
+                           b'ssh://git@github.com/']:
+                if out.startswith(prefix):
```

Comment:
Will do (in an hour or so, when I'm not typing on a phone). Thanks for the quick fix on the mne import issue BTW